code/__DEFINES/obj_flags.dm

```diff
@@ -68,7 +68,8 @@
 #define PLASMAMAN_PREVENT_IGNITION (1<<18)
 /// Usable as casting clothes by wizards (only matters for suits and headwear)
 #define CASTING_CLOTHES (1<<19)
-
+/// Allows for masks to use tanks on adjacentadjecent tiles.
```

Comment:
encapsulate the comment and define with //SKYRAT EDIT comments as usual if you leave this non-modular

Reply:
Golden recommended that I keep it in the same file as the TG defines.